lib/v8-to-istanbul.js

```diff
@@ -102,10 +102,7 @@ module.exports = class V8ToIstanbul {
   }
 
   destroy () {
-    if (this.sourceMap) {
-      this.sourceMap.destroy()
-      this.sourceMap = undefined
-    }
+    this.sourceMap = undefined
```

Comment:
is this needed now? `destroy` was added in #161, and I don't think it does much now? can keep the method around to avoid a breaking change I guess, but beyond that?

Reply:
No, I'll open a second PR to address.